src/DIRAC/FrameworkSystem/Agent/ComponentSupervisionAgent.py

```diff
@@ -565,8 +569,8 @@ def _checkServiceURL(self, serviceName, options):
     module = options['Module']
     self.log.info('Checking URLs for %s/%s' % (system, module))
     urlsConfigPath = PathFinder.getSystemURLSection(
```

Comment:
```suggestion
    urlsConfigPath = Path.cfgPath(PathFinder.getSystemURLSection(
```

Reply:
it is not necessary, it's already done by `PathFinder`